lib/fictium/configurations/configuration.rb

```diff
@@ -1,10 +1,62 @@
 module Fictium
   class Configuration
-    attr_accessor :exporters
+    VOWEL = /[aeiou]/i.freeze
```

Comment:
If you are only create this constant for this file, you should add this line
```rb
VOWEL = /[aeiou]/i.freeze
private_constant :VOWEL
```
Also, delete VERB_TERMINATOR unused constant
[private_constant](http://www.chrisrolle.com/en/blog/private-constants-in-ruby)